copy/copy.go

```diff
@@ -170,6 +170,10 @@ type Options struct {
 	ForceManifestMIMEType string
 	ImageListSelection    ImageListSelection // set to either CopySystemImage (the default), CopyAllImages, or CopySpecificImages to control which instances we copy when the source reference is a list; ignored if the source reference is not a list
 	Instances             []digest.Digest    // if ImageListSelection is CopySpecificImages, copy only these instances and the list itself
+	// ImageDestination is a preset types.ImageDestination object. If not nil, the `destRef` will be ignored.
+	ImageDestination types.ImageDestination
+	// ImageSource is a preset types.ImageSource object. If not nil, the `srcRef` will be ignored.
+	ImageSource types.ImageSource
```

Comment:
This is not really necessary; it’s, in the extreme, possible to create a do-nothing `ImageReference` that just returns the right `ImageDestination`/`ImageSource`.

But doing so would also more starkly expose the need to deal with names/identities (if nothing else, so that error messages from `copy.Image` are meaningful). I do think that needs to happen — the multi* objects should be a way to get (lifetime-limited, but otherwise pretty ordinary) `ImageReference` values.